src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF_8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device_width, user_scalable=no, initial_scale=1.0, maximum_scale=1.0, minimum_scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card">
+      <div class="card__pc">
+        <div class="card__img"></div>
+        <div class="card__descr">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__prodcode">
+          Product code: 195434
+        </div>
+      </div>
+
+
+      <div class="card__reviews">
+
+        <div class="card__stars">
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+          <div class="card__stars-star"></div>
+        </div>
+
+        <div class="card__label">
```

Comment:
```suggestion
        <p class="card__label">
```
use `p` tag for text